reactor-core/src/main/java/reactor/util/Loggers.java

```diff
@@ -442,18 +442,45 @@ public void error(String msg, Throwable t) {
 		}
 
 		@Nullable
-		final String format(@Nullable String from, @Nullable Object... arguments){
-			if(from != null) {
+		String format(@Nullable String from, @Nullable Object... arguments) {
+			return format(from, false, arguments);
+		}
+
+		@Nullable
+		private String format(@Nullable String from, boolean skipLast, @Nullable Object... arguments) {
```

Comment:
> I added the skipLast parameter to the format(...) method. Since Varargs must be placed at the end, I added skipLast as a middle parameter.
Additionally, I overloaded the existing method.

Would it work if instead of `Object...` you used `Object[]` ? Since this is an internal API it should be totally fine with the usages here. Then the order of arguments will be more appropriate IMO.

Reply:
I have changed it to use `Object[]` instead.